docker/entrypoint.sh

```diff
@@ -3,7 +3,7 @@
 set -euo pipefail
 cd /
 
-usermod -u "$(stat -c %u /data/www)" apache
+usermod -u "$(sudo -u apache stat -c %u /data/www)" apache
```

Comment:
I don't know what to do here run `usermod -u "$(stat -c %u /data/www)" apache` only if not on mac?

Reply:
I think it should still work on non-macOS if you use `sudo` like this. Let's try it, if it breaks when I work with it we'll see again.